### DISCUSSION TITLE
Deployment issue on the Cloud platform

### DISCUSSION THREAD

**Discussion by Masterchenyong** [milvus-io/milvus#3419] [Q&A and General discussion] [answered]
Question:

I deployed the basic milvus container on the cloud server and then used the IP port connection to flash data in so that the performance of the operation could reach 10ms, but I directly flashed the data locally, marked it into a large image and deployed it to the server, performance When the delay reaches 100ms, the amount of memory and the number of CPU cores given by the two methods are the same, the preload is also changed to "*", and the amount of data is 1 million. The data swiped online is also stored in the container.

我在云服务器上部署基础的milvus容器，然后通过ip端口连接刷数据进去，这样操作的性能可以达到10ms，但是我直接将数据在本地刷好，打成一个大的镜像再部署到服务器，性能延迟到了100ms，两种方法给的内存数量和cpu核数都是一样的，预加载也改成了"*"，数据量都是100万。线上刷进去的数据也是在保存在容器里面。

**Comment by gujun720** [comment, marked as answer]
Hello, Masterchenyong

English is the **only communication language** in the Discussions of the Milvus project, as it is an incubation project at the LF AI foundation.

You can use Google translation to convert the question into English. I have done it for you this time.

Going forward please follow the [guidelines of community discussion](https://github.com/milvus-io/milvus/discussions/3420).


**Question:**

I deployed the basic milvus container on the cloud server and then used the IP port connection to flash data in so that the performance of the operation could reach 10ms, but I directly flashed the data locally, marked it into a large image and deployed it to the server, performance When the delay reaches 100ms, the amount of memory and the number of CPU cores given by the two methods are the same, the preload is also changed to "*", and the amount of data is 1 million. The data swiped online is also stored in the container.

**Comment by gujun720** [comment]
@Masterchenyong could you elaborate on what kind of cloud platform you are using?

In these 2 situations, were you using the same kind of disk to store the data?
